src/styles/header.scss

```diff
@@ -0,0 +1,62 @@
+@import 'variables';
+
+body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+}
+
+.header {
+  padding: 0 $header-padding;
```

Comment:
The variable `$header-padding` should be defined in the 'variables' file or elsewhere before it is used here. Ensure that it is correctly imported or defined.